src/tests/stencil_tests.cpp

```diff
@@ -0,0 +1,169 @@
+#include "stencil_tests.h"
+
+#include <pbkit/pbkit.h>
+
+#include "../shaders/precalculated_vertex_shader.h"
```

Comment:
nit/FYI the makefile adds `tests` as an include path so you shouldn't need to do a relative include.

Reply:
Fixed